modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/url/UrlContextHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.enonic.xp.portal.impl.url;
+
+import com.enonic.xp.context.ContextAccessor;
+
+public class UrlContextHelper
```

Comment:
Entire class to deduplicate one line of code. Me not like

Reply:
I am not agree with you. That is used at least in two places in AttachentUrlBuilder and ImageUrlBuilder.